apps/web/src/hooks/useDropUpload.ts

```
@@ -0,0 +1,334 @@
+/** How long a settled row stays on screen before it retires itself. */
+const SETTLED_ROW_MILLIS = 1500;
+
+/** Where an upload has got to; `staging` is the only rung the engine does not name. */
+export type UploadPhase =
+  | 'staging'
+  | 'queued'
+  | 'uploading'
+  | 'uploaded'
+  | 'stalled'
+  | 'cancelled'
+  | 'failed';
```

ESLint

[error] Replace `|·'staging'⏎··|·'queued'⏎··|·'uploading'⏎··|·'uploaded'⏎··|·'stalled'⏎··|·'cancelled'⏎·` with `'staging'·|·'queued'·|·'uploading'·|·'uploaded'·|·'stalled'·|·'cancelled'`

(prettier/prettier)